packages/web3-eth-personal/package.json

```diff
@@ -11,17 +11,25 @@
     "scripts": {
         "build": "rollup -c",
         "dev": "rollup -c -w",
-        "test": "jest"
+        "test": "jest",
+        "dtslint": "dtslint types --onlyTestTsNext"
     },
+    "types": "types",
     "dependencies": {
+        "typescript": "^3.3.0-dev.20181213",
         "web3-core": "1.0.0-beta.36",
         "web3-core-helpers": "1.0.0-beta.36",
         "web3-core-method": "1.0.0-beta.36",
+        "web3-eth": "1.0.0-beta.36",
```

Comment:
you can't add this dependency in here, this will cause a change to the dependency tree which is not allowed for this version of web3. If you are using types from web3.eth types we need to move them to web3-core so they can be shared.

Reply:
okay, so for now should I just define the types again inside Personal?